cmd/kpod/main.go

```diff
@@ -58,11 +58,16 @@ func main() {
 		waitCommand,
 	}
 	app.Before = func(c *cli.Context) error {
-		logrus.SetLevel(logrus.ErrorLevel)
-		if c.GlobalBool("debug") {
+		if logLevel := c.GlobalString("log-level"); logLevel != "" {
+			level, err := logrus.ParseLevel(logLevel)
+			if err != nil {
+				return err
+			}
+
 			debug = true
```

Comment:
You should still check for the "debug" string, before setting debug=true.

Reply:
fixed